### DISCUSSION TITLE
Elsa 3 Nuget Source error

### DISCUSSION THREAD

**Discussion by dodyg** [elsa-workflows/elsa-core#3664] [Q&A] [answered]
I am following the instruction here `https://v3.elsaworkflows.io/docs/installation/packages` and set the following `nuget.config`

```xml
<?xml version="1.0" encoding="utf-8"?>
<configuration>
  <packageSources>
    <clear />
    <add key="NuGet official package source" value="https://api.nuget.org/v3/index.json" />
    <add key="Elsa 3 preview" value="https://pkgs.dev.azure.com/elsa-workflows/3cbdb983-acb6-4ba7-b862-f9e3cbd4e213/_packaging/Packages/nuget/v3/index.json" />
  </packageSources>
</configuration>
```

and I got `error:   Response status code does not indicate success: 404 (Not Found - The feed with ID 'Packages' doesn't exist. (DevOps Activity ID: E75CC271-669F-4024-8505-E668690C5136)).` when executing `dotnet add package Elsa`.

https://pkgs.dev.azure.com/elsa-workflows/3cbdb983-acb6-4ba7-b862-f9e3cbd4e213/_packaging/Packages/nuget/v3/index.json

**Comment by sfmskywalker** [comment, marked as answer]
Sorry about that, I changed the feed name but then forgot to update the documentation. I just pushed and update to the documentation. The new feed URL is: https://pkgs.dev.azure.com/elsa-workflows/3cbdb983-acb6-4ba7-b862-f9e3cbd4e213/_packaging/Elsa3/nuget/v3/index.json

Thanks for reporting!

**Reply by theafricaninternetgroup** [reply]
Thanks I was getting  the same error

**Comment by agriffard** [comment]
See PR https://github.com/elsa-workflows/elsa-documentation/pull/5
